src/include/settings/settings_defs.h

```diff
@@ -316,4 +316,12 @@ SETTING_bool(
     true,
     noisepage::settings::Callbacks::NoOp
 )
+
+SETTING_bool(
+    use_messenger,
```

Comment:
I think the convention we standardized on is `<component>_enable` for settings name.